webapp/components/User/User.vue

```diff
@@ -148,6 +149,13 @@ export default {
       this.user.followedByCount = followedByCount
       this.user.followedByCurrentUser = followedByCurrentUser
     },
+    async block(user) {
+      await this.$apollo.mutate({ mutation: Block(), variables: { id: user.id } })
+      this.$toast.success(user.name + 'geblockt')
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-11-04T07:06:29Z" title="Monday, November 4th 2019, 8:06:29 am +01:00">Nov 4, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -148,6 +149,13 @@ export default {
       this.user.followedByCount = followedByCount
       this.user.followedByCurrentUser = followedByCurrentUser
     },
+    async block(user) {
+      await this.$apollo.mutate({ mutation: Block(), variables: { id: user.id } })
+      this.$toast.success(user.name + 'geblockt')
```


can we use a translation string instead of `geblockt`?